src/main/webapp/WEB-INF/tags/admin/search/instructorRow.tag

```diff
@@ -35,13 +35,21 @@
         <c:if test="${not empty instructor.viewRecentActionsId}">
             <form method="post" target="blank" action="<%=Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE%>">
                 <button type="submit" id="${instructor.viewRecentActionsId}_recentActions"
-                        class="btn btn-link btn-xs recentActionButton">
+                        class="btn btn-link btn-xs optionButton">
                         
                     <span class="glyphicon glyphicon-zoom-in"></span>View Recent Actions
                 </button>
                 <input type="hidden" name="filterQuery" value="${instructor.viewRecentActionsId}">
                 <input type="hidden" name="courseId" value="${instructor.courseId}">
             </form>
+            <c:if test="${not empty instructor.googleId}">
+                <form method="post" target="blank" action="<%=Const.ActionURIs.ADMIN_ACCOUNT_MANAGEMENT_PAGE%>">
```

Comment:
@bamboo3250, personally I think using GET instead of POST for this page is much more suitable, although that may result in this being inconsistent with the other links surrounding this.